synapse/storage/databases/main/receipts.py

```diff
@@ -170,8 +170,8 @@ def get_last_receipt_for_user_txn(
         receipt_types: Collection[str],
     ) -> Optional[Tuple[str, int]]:
         """
-        Fetch the event ID and stream_ordering for the latest receipt in a room
-        with one of the given receipt types.
+        Fetch the event ID and stream_ordering for the latest unthreaded receipt
+        in a room with one of the given receipt types.
```

Comment:
How do I get a threaded receipt?

Reply:
You don't need to, the callers of this function always want the unthreaded one.

Reply:
I more meant it in terms https://github.com/matrix-org/synapse/pull/13877#discussion_r981990073 - What function to use if I want the threaded world receipt?

Reply:
> I more meant it in terms [#13877 (comment)](https://github.com/matrix-org/synapse/pull/13877#discussion_r981990073) - What function to use if I want the threaded world receipt?

There is no function to get that. The functions which care about it do it internally via sub-queries.